### PR TITLE
[Semi modular] Fixes chronosuit sprites being errors, adds digi sprites to honkerative and chronosuits

### PR DESCRIPTION
**Pull request by Ebin-Halcyon**
## About The Pull Request

Hey it works.
image
image

## How This Contributes To The Skyrat Roleplay Experience

sprites work :)

## Changelog

:cl:
imageadd: Chrono suit + Honkerative digi sprites
fix: Chrono suits are no longer sent to the 4th dimension and now properly have worn sprites
/:cl:

### CODE REVIEW

File: code/modules/clothing/outfits/standard.dm [hunk 1]
```diff
@@ -421,12 +421,12 @@
...
 	mask = /obj/item/clothing/mask/breath
-	back = /obj/item/mod/control/pre_equipped/chrono
+	back = /obj/item/mod/control/pre_equipped/timeline
 
 /datum/outfit/chrono_agent/post_equip(mob/living/carbon/human/agent, visualsOnly)
 	. = ..()
-	var/obj/item/mod/control/mod = agent.back
-	var/obj/item/mod/module/eradication_lock/lock = locate(/obj/item/mod/module/eradication_lock) in mod.modules
+	var/obj/item/mod/control/pre_equipped/timeline = agent.back
+	var/obj/item/mod/module/eradication_lock/lock = locate(/obj/item/mod/module/eradication_lock) in timeline.modules
```
Comment (GoldenAlpharex) [inline code/modules/clothing/outfits/standard.dm:429]: Are those how it was meant to be? I might've done goofed when fixing the mirror, to be fair.

File: code/modules/mod/mod_types.dm [hunk 2]
```diff
@@ -283,8 +283,8 @@
 		/obj/item/mod/module/holster,
 	)
 
-/obj/item/mod/control/pre_equipped/chrono
-	theme = /datum/mod_theme/chrono
+/obj/item/mod/control/pre_equipped/timeline
+	theme = /datum/mod_theme/timeline
```
Comment (GoldenAlpharex) [inline code/modules/mod/mod_types.dm:287]: Man sometimes I should test that kind of mirror.

But I forgor 💀

File: code/modules/clothing/outfits/standard.dm (hunk 1 above)
Comment (Ebin-Halcyon) [reply code/modules/clothing/outfits/standard.dm:429]: Yuh, originally was just called 'chrono' while in files it was called timeline. Just had to be corrected as such.

**Review by GoldenAlpharex — APPROVED** [review]
Yes.

**Pull request merged**